config/rbac/role.yaml

```diff
@@ -142,6 +142,32 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
```

Comment:
New resources must also be added to the Helm chart's  RBAC!

Reply:
done